File: exercicios/Python/1064/1064.py
n1 = float(input())
n2 = float(input())
n3 = float(input())
n4 = float(input())
n5 = float(input())
n6 = float(input())

p = 0;
m = 0;

v =[n1,n2,n3,n4,n5,n6]

for n in v:
    if n > 0:
        p = p + 1;
        m = m + (n)

print('{} valores positivos'.format(p))
print('{:.1f}'.format(m/p))

File: exercicios/R/1064/1064.r
input = file('stdin','r')

qt <- c(1:6)

media = as.double(0.0)
cont = as.integer(0)
soma = as.double(0.0)

for (i in qt) {
   v = as.double(readLines(input,n=1))

   if (v > 0) {
      soma = soma + v
      cont = cont + 1
   }
}
media = soma / cont
write(sprintf("%d valores positivos",cont),'')
write(sprintf("%.1f",media),'')